src/Raven.Studio/wwwroot/App/views/database/settings/databaseSettings.html

```diff
@@ -0,0 +1,253 @@
+            <button class="btn btn-default dropdown-toggle filter-entries" data-toggle="dropdown">
+                <span data-bind="visible: groupToShow() === 'allEntries'"><span>All Entries</span></span>
+                <span data-bind="visible: groupToShow() === 'databaseEntries'"><span>Database Entries</span></span>
+                <span data-bind="visible: groupToShow() === 'customizedDatabaseEntries'"><span>Customized Database Entries</span></span>
+                <span class="caret"></span>
+            </button>
+            <ul class="dropdown-menu">
+                <li title="Show all entries per database & server-wide"><a href="#" data-bind="click: _.partial(setGroupToShow, 'allEntries')"><span>All Entries</span></a></li>
+                <li title="Show only entries per this database"><a href="#" data-bind="click: _.partial(setGroupToShow, 'databaseEntries')"><span>Database Entries</span></a></li>
+                <li title="Show entries that are customized per this database"><a href="#" data-bind="click: _.partial(setGroupToShow, 'customizedDatabaseEntries')"><span>Customized Database Entries</span></a></li>
+            </ul>
+        </div>
+        <div class="filter-entries">
+            <input class="form-control" placeholder="Filter Keys" data-bind="textInput: filterKeys" />
+        </div>
+        <div>
+            <button class="btn btn-default" data-bind="click: refresh, css: { 'btn-spinner': spinners.refresh }, disable: spinners.refresh() || spinners.save()" title="Reload settings">
+                <i class="icon-refresh"></i><span>Refresh</span>
+            </button>
+        </div>
+    </div>
+    <div class="flex-horizontal flex-noshrink summary-buttons">
+        <div class="flex-separator"></div>
+        <div data-bind="visible: viewMode() === 'summaryMode'">
+            <button class="btn btn-sm btn-default" data-bind="click: switchToEditMode" title="Edit the Database Settings">
+                <i class="icon-edit"></i><span>Edit Entries</span>
+            </button>
+        </div>
+        <div data-bind="visible: viewMode() === 'editMode', enable: !isSaveEnabled()">
+            <button class="btn btn-sm btn-default" data-bind="click: exitEditMode" title="View all effective values in a condensed view">
+                <i class="icon-table"></i><span>View Summary</span>
+            </button>
+        </div>
+    </div>
+    <div class="flex-horizontal flex-stretch-items absolute-fill content-margin entries-grid" data-bind="visible: viewMode() === 'editMode'">
+        <div class="flex-vertical">
+            <div class="categories-list-container flex-vertical flex-grow" style="position:relative;">
+                <virtual-grid class="resizable virtual-grid" params="controller: categoriesGridController, emptyTemplate: 'db-settings-no-entries-loaded-template'"></virtual-grid>
+            </div>
+        </div>
+        <div class="flex-grow flex-vertical slidein-style">
```

Comment:
do we need slidein ?

Reply:
Well, it gives **.25s** **ease-in** delay when switching between the 2 grids
Defined as: `@slidein-properties: slidein .25s cubic-bezier(0, 0, 0, 1);`
If I remove this class it is just a bit more 'jumpy'